lib/health_checks/rails.rb

```diff
@@ -16,17 +19,39 @@ def rails
 
   private
 
-  class RedisCheck < OkComputer::Check
-    def check
-      sidekiq_response = ::Sidekiq.redis(&:ping)
+  module Logger
+    module_function
 
-      if sidekiq_response == 'PONG'
-        mark_message 'Connected to Redis'
-        return
-      end
+    def error(error)
+      Rails.logger.error "Error: #{error}"
+    end
+  end
 
+  class CustomMongoidCheck < OkComputer::MongoidCheck
+    def initialize(db)
+      @db_name = db[:name]
+      client_name = "#{@db_name}_rails_health_check"
+      self.session = MongoidCustomClient.create(client_name, db[:hosts])
+    end
+
+    def check
+      mongodb_name
+      mark_message "Connected to mongodb #{@db_name}"
+    rescue => e
+      mark_failure
+      mark_message e.message
+      Logger.error("#{e}, Database Name: #{@db_name}")
```

Comment:
Why do you log a different message than the one you responded with?

Reply:
#mark_message adds the required information. When the check is registered, the db name is passed and later it is printed. No need to repeat it.

Reply:
:+1: